src/v3/src/types/schema.ts

```diff
@@ -171,34 +171,58 @@ export interface UISchemaElement {
   noTranslate?: boolean;
 }
 
-export interface UISchemaLayout {
+/**
+ * Parent interface for all Layout types with common properties
+ */
+interface Layout {
+  type: string;
+  key?: string;
+  elements: unknown[];
+  options?: Record<string, unknown>;
+}
```

Comment:
Not sure if this is really helpful so feel free to veto if you think this parent type is more harmful than helpful